charts/pip-data-management/values.yaml

```diff
@@ -26,3 +26,4 @@ java:
   #        - name: AppInsightsInstrumentationKey
   #          alias: azure.application-insights.instrumentation-key
   environment:
+    RUN_DB_MIGRATION_ON_STARTUP: true
```

Comment:
this shouldn't be enabled by default in values.yaml only in values.preview.template.yaml